media-nep/src/main/java/nl/vpro/nep/service/impl/NEPGatekeeperServiceImpl.java

```diff
@@ -173,23 +175,28 @@ public WorkflowExecution transcode(
 
     @NonNull
     @Override
-    @SneakyThrows
     public Iterator<WorkflowExecution> getTranscodeStatuses(
         @Nullable String mid,
         @Nullable StatusType status,
         @Nullable Instant from,
-        @Nullable Long limit) {
+        @Nullable Long limit) throws NEPException {
         final int batchSize = pageSize;
-        URIBuilder builder = new URIBuilder(getWorkflowsEndPoint());
+        URIBuilder builder = null;
+        try {
+            builder = new URIBuilder(getWorkflowsEndPoint());
+        } catch (URISyntaxException e) {
+            throw new NEPException(e, "Exception from a NEP function call");
+        }
```

Comment:
Dit ljkt me dan toch weer een beetje onzin. Als dit gebeurt dan is het niet de schuld van NEP.